tests/test_server.py

```diff
@@ -334,6 +330,54 @@ def test_train_internal_error(rasa_app: SanicTestClient):
     assert response.status == 500
 
 
+@pytest.fixture
+def formbot_data():
+    return dict(
+        domain="examples/formbot/domain.yml",
+        config="examples/formbot/config.yml",
+        stories="examples/formbot/data/stories.md",
+        nlu="examples/formbot/data/nlu.md",
+    )
+
+
+def test_train_status(rasa_server, formbot_data):
+    with ExitStack() as stack:
+        payload = {
+            key: stack.enter_context(open(path)).read()
+            for key, path in formbot_data.items()
+        }
+
+    from multiprocessing import Process, Manager
```

Comment:
should probably go to the top of the file or is there a reason to scope it within this test?